pyproject.toml

```diff
@@ -13,6 +13,7 @@ readme = "README.md"
 python = "^3.9"
 pyyaml = "^6.0"
 google-cloud-logging = "^3.5.0"
+colorlog = "^6.7.0"
```

Comment:
If this is for testing purposes, shouldn't it be a dev dependency? We don't want this to be used in production, right?

Reply:
Changed colorlog to a dev dependency